internal/cli/server/config.go

```diff
@@ -417,9 +421,9 @@ func DefaultConfig() *Config {
 			PriceLimit:   1,
 			PriceBump:    10,
 			AccountSlots: 16,
-			GlobalSlots:  4096,
+			GlobalSlots:  131072,
```

Comment:
@manav2401 please confirm these numbers for globalSlots and globalQueue. I guess we decreased them to 32768.